src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operation.BalanceOperationHandler;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseOperationHandler;
+import core.basesyntax.strategy.operation.ReturnOperationHandler;
+import core.basesyntax.strategy.operation.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static final int TEST_FRUIT_AMOUNT_ONE = 10;
+    private static final int TEST_FRUIT_AMOUNT_TWO = 20;
+    private static final String TEST_FRUIT_TYPE_ONE = "avocado";
+    private static final String TEST_FRUIT_TYPE_TWO = "papaya";
+    private static final boolean GET_OPERATION_HANDLER = true;

```

Comment:
Think about local variables, I think i can use variable one time you can use local

Reply:
I provided these variables here to avoid "magic" numbers and values.